source/trackers.json

```diff
@@ -238,6 +244,7 @@
         }
     },
     "trackerDomains": {
+        "telstra.com.au": "telstra",
```

Comment:
telstra.com?

Reply:
Telstra is the largest telco and uses .com.au.

All .com domains redirect to .com.au's.